circle.yml

```diff
@@ -1,7 +1,7 @@
 machine:
   post:
   - cd $HOME && git clone --depth 1 -v git@github.com:clever/ci-scripts.git && cd ci-scripts && git show --oneline -s
-  - $HOME/ci-scripts/circleci/golang-install 1.8
+  - $HOME/ci-scripts/circleci/golang-install 1.9
```

Comment:
why upgrade?

Reply:
That's what's on my computer. Is there a reason we shouldn't?

Reply:
Since we run 1.8 everywhere else in production. 1.9 is under 1 month old so it feels a little pre-mature to move over.

Reply:
k i'll keep it as is